certified-connectors/Nodefusion Portal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,166 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Nodefusion Portal",
+    "description": "A Nodefusion Portal Custome Connector is used for various actions, such as opening support case, or subscription activation",
+    "contact": {
+      "name": "Nodefusion",
+      "url": "https://www.nodefusion.com",
+      "email": "nodefusion@nodefusion.com"
+    },
+    "license": {
+      "name": "MIT",
+      "url": "http://opensource.org/licenses/MIT"
+    },
+    "version": "5.1.0.4732"
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.nodefusion.com"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.nodefusion.com/terms"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "IT Operations;Website"
+    }
+  ],
+  "host": "3proportal-functions.azurewebsites.net",

```

Comment:
@hkusulja Please note that we do not certify a connector that has a development or staging host URL.

Reply:
This is production host url